gradoop-examples/src/main/java/org/gradoop/examples/benchmark/grouping/GroupingBenchmark.java

```diff
@@ -0,0 +1,459 @@
+ */
+
+package org.gradoop.examples.benchmark.grouping;
+
+import com.google.common.collect.Lists;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.io.FileUtils;
+import org.apache.flink.api.common.ProgramDescription;
+import org.gradoop.examples.AbstractRunner;
+import org.gradoop.model.impl.LogicalGraph;
+import org.gradoop.model.impl.operators.grouping.Grouping;
+import org.gradoop.model.impl.operators.grouping.GroupingStrategy;
+import org.gradoop.model.impl.operators.grouping.functions.aggregation.CountAggregator;
+import org.gradoop.model.impl.operators.grouping.functions.aggregation.MaxAggregator;
+import org.gradoop.model.impl.operators.grouping.functions.aggregation.MinAggregator;
+import org.gradoop.model.impl.operators.grouping.functions.aggregation.PropertyValueAggregator;
+import org.gradoop.model.impl.pojo.EdgePojo;
+import org.gradoop.model.impl.pojo.GraphHeadPojo;
+import org.gradoop.model.impl.pojo.VertexPojo;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.regex.Pattern;
+
+/**
+ * A dedicated program for parametrized graph grouping benchmark.
+ */
+public class GroupingBenchmark extends AbstractRunner
+  implements ProgramDescription {
+
+  /**
+   * Option to declare path to input graph
+   */
+  public static final String OPTION_INPUT_PATH = "i";
+  /**
+   * Option to declare path to output graph
+   */
+  public static final String OPTION_OUTPUT_PATH = "o";
+  /**
+   * Vertex grouping key option
+   */
+  public static final String OPTION_VERTEX_GROUPING_KEY = "vgk";
+  /**
+   * Edge grouping key option
+   */
+  public static final String OPTION_EDGE_GROUPING_KEY = "egk";
+  /**
+   * Use vertex label option
+   */
+  public static final String OPTION_USE_VERTEX_LABELS = "uvl";
+  /**
+   * Use edge label option
+   */
+  public static final String OPTION_USE_EDGE_LABELS = "uel";
+  /**
+   * Path to CSV log file
+   */
+  public static final String OPTION_CSV_PATH = "csv";
+  /**
+   * Used vertex aggregator (min, max, count, none)
+   */
+  private static final String OPTION_VERTEX_AGGREGATION = "vagg";
+  /**
+   * Used vertex aggregation keys
+   */
+  private static final String OPTION_VERTEX_AGGREGATION_KEYS = "vak";
+  /**
+   * Used Vertex aggregation result keys
+   */
+  private static final String OPTION_VERTEX_AGGREGATION_RESULT_KEYS = "vark";
+  /**
+   * Used Edge aggregator (min, max, count, none)
+   */
+  private static final String OPTION_EDGE_AGGREGATION = "eagg";
+  /**
+   * Used vertex aggregation keys
+   */
+  private static final String OPTION_EDGE_AGGREGATION_KEYS = "eak";
+  /**
+   * Used Vertex aggregation result keys
+   */
+  private static final String OPTION_EDGE_AGGREGATION_RESULT_KEYS = "eark";
+  /**
+   * Used VertexKey for grouping
+   */
+  private static String VERTEX_GROUPING_KEYS;
```

Comment:
if they are not final, they are no constants, so no need to write them all upper case